Framework/Kernel/inc/MantidKernel/PropertyWithValue.tcc

```diff
@@ -399,12 +399,13 @@ void PropertyWithValue<TYPE>::replaceValidator(IValidator_sptr newValidator) {
 template <typename TYPE>
 std::string
 PropertyWithValue<TYPE>::setValueFromProperty(const Property &right) {
-  auto prop = dynamic_cast<const PropertyWithValue<TYPE> *>(&right);
-  if (!prop) {
-    return "Could not set value: properties have different type.";
-  }
-  m_value = prop->m_value;
-  return "";
+  
+  if (auto prop = dynamic_cast<const PropertyWithValue<TYPE> *>(&right)) {
+    m_value = prop->m_value;
+    return "";
+  }else{
```

Comment:
This does not look clang-formatted; I suppose it's skipped because of the `.tcc` extension. Can you have a look?

Reply:
the only difference in the file appears to be a space either side of the else, I'll add it now

Reply:
yes, that's what I spotted; but I was worried why it wasn't caught in the previous run of clang-format.

Reply:
I did the clang formatting locally this time rather than applying a patch, I imagine your likely right about it be because it was a .tcc file that the patch didn't cover it

Reply:
@gvardany Good catch. I think the job must not be searching for `.tcc` files. We should fix that.